src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -525,7 +525,7 @@ protected override void StopProcessing()
         private CustomSerialization _serializer;
 
         /// <summary>
-        ///Memory Stream used for  serialization
+        ///Memory Stream used for serialization
```

Comment:
Please add space before `Memory`.
It seems we should set final point.

Reply:
Ok